integration-tests/tagsoup/pom.xml

```diff
@@ -69,6 +74,12 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
It would be nice to see a comment here explaining why we skip the enforcer, ideally with a link to an issue.